locale/es.po

```diff
@@ -885,17 +885,50 @@ msgstr ""
 msgid "function missing required positional argument #%d"
 msgstr ""
 
+<<<<<<< HEAD
 #: py/bc.c:260
 msgid "function missing required keyword argument '%q'"
 msgstr ""
+||||||| merged common ancestors
+#: ports/atmel-samd/common-hal/analogio/AnalogOut.c:53
+msgid "AnalogOut not supported on given pin"
+msgstr ""
+=======
+#: ports/atmel-samd/common-hal/analogio/AnalogOut.c:53
+msgid "AnalogOut not supported on given pin"
+msgstr "AnalogOut no es soportado por el pin dado"
```

Comment:
Looks like you missed some merge markers. Here you'll want to delete the empty "AnalogOut not supported on given pin" entry and the markers.

Reply:
Hi, fixed with 0aef043, please let me know if i missed something.